src/Microsoft.Health.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -220,10 +232,26 @@ public override void Set(SqlDataRecord record, int ordinal, decimal value)
     public class FloatColumn : Column<double>
     {
         public FloatColumn(string name, byte precision)
-            : base(name, SqlDbType.Float, false, precision, 0)
+            : base(name, SqlDbType.Float, false, GetLength(precision), precision, 0, 0, SqlCompareOptions.None, null)
         {
         }
 
+        private static long GetLength(byte precision)
```

Comment:
Any way to abstract this to where we're not copying it? Maybe a separate static class in this file?